client/look-and-feel/css/src/Stepper/Stepper.scss

```
@@ -1,12 +1,15 @@
+@use "../common/common.scss" as common;
+
 .af-stepper-client {
   display: flex;
   width: 100%;
   flex-direction: column;
+  gap: common.rem(16);
```

On c'est dit de limité le plus possible l'utilisation de sass au profit du css natif

```suggestion
  gap: calc(16rem/16);
```

---

Fixed in : https://github.com/AxaFrance/design-system/pull/894/commits/3231c1d40197873ad0dbcdeb056818f93b14196d